iOS/SlackTextViewController/External-Dependency-Info.txt

```diff
@@ -0,0 +1,39 @@
+THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
+Do not translate or localize
+
+Xamarin Components for iRate Rating Prompt incorporates third party 
+material from the projects listed below. The original copyright notice and 
+the license under which Microsoft received such third party material are set 
+forth below.  Microsoft reserves all other rights not expressly granted, 
+whether by implication, estoppel or otherwise.
+
+########################################
+# SlackTextViewController
+# https://github.com/slackhq/SlackTextViewController
```

Comment:
Please check the repo for " Copyright (c) 2013 Lukas Kubanek"

If we are using code under that copyright please add it to this file.